docs/user/guides/build/index.rst

```diff
@@ -0,0 +1,34 @@
+How-to guides: build process
+============================
+
+⏩️ :doc:`Setup build notifications and webhooks </guides/build-notifications>`
+    Build notifications can alert you when your builds fail so you can take immediate action.
+    In this guide,
+    you will learn how to get notified through various available channel integrations,
+    including email and chat.
+
+⏩️ :doc:`Configure pull request builds </guides/pull-requests>`
+    Have your documentation built and access a preview for every :doc:`pull request builds </pull-requests>`.
+
+⏩️ :doc:`Use custom environment variables </guides/environment-variables>`
+    Extra environment variables, for instance secrets, may be needed in the build process and can be defined from the project's :term:`dashboard`.
+
+⏩️ :doc:`Manage versions automatically </guides/automation-rules>`
+    Automating your versioning on Read the Docs means you only have to handle your versioning logic in Git.
+    Learn how to define rules to automate creation of new versions on Read the Docs,
+    entirely using your Git repository's version logic.
+
+
+.. seealso:
+
+   :doc:`/builds`
+      An overview and introduction to the build process.
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   Setup build notifications and webhooks </guides/build-notifications>
```

Comment:
I still think we should just make these the titles of the pages, instead of having to hack them everywhere...

This just seems like a massive amount of unnecessary work to rewrite them every time we use them in both the listing and TOCTree view. 

I think users will understand they are how to guides, even if they don't start with `how to`.

Reply:
> I still think we should just make these the titles of the pages, instead of having to hack them everywhere...

I disagree with "massive" and "hack", I feel quite relaxed about it :)

Now it's also done for all these titles and put into the Style Guide. So I'd consider it job done. The convention is repeated a lot of times, so I hope we'll remember it automatically :)

The thing is that we _could_ compromise in this one case with titles and navigation labels. But that doesn't fix our need to have a pattern for introduction sentences and to repeat similar structures for our feature reference.

Static site generators are great at this, it's just because Sphinx can't do it yet it seems.